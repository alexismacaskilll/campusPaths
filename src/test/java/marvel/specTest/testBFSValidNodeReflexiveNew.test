# Test the Find Path from a node to itself and the node is in the graph

# Load a graph from tsv file that contains two heros in same book and one hero in a
# different book.
LoadGraph graph1 threeHerosTwoBooksNew.tsv

# Find shortest path from hero 1 to hero 1
FindPath graph1 hero_1 hero_1