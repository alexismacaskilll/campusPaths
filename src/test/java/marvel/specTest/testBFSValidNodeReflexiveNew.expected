# Test the Find Path from a node to itself and the node is in the graph

# Load a graph from tsv file that contains two heros in same book and one hero in a
# different book.
loaded graph graph1

# Find shortest path from hero 1 to hero 1
path from hero 1 to hero 1: